File: src/main/java/com/pengrad/telegrambot/model/request/ReplyKeyboardRemove.java
package com.pengrad.telegrambot.model.request;

/**
 * Stas Parshin
 * 23 November 2016
 */
public class ReplyKeyboardRemove extends Keyboard {

    private final boolean remove_keyboard = true;
    private final boolean selective;

    public ReplyKeyboardRemove() {
        this(false);
    }

    public ReplyKeyboardRemove(boolean selective) {
        this.selective = selective;
    }

}

File: core/src/commonMain/kotlin/com/pengrad/telegrambot/model/request/ReplyKeyboardRemove.kt
package com.pengrad.telegrambot.model.request

import java.io.Serializable

/**
 * Stas Parshin
 * 23 November 2016
 */
class ReplyKeyboardRemove @JvmOverloads constructor(private val selective: Boolean = false) : Keyboard(), Serializable {

    private val remove_keyboard = true

    companion object {
        private const val serialVersionUID = 0L
    }

}
